src/Mono.Android/Android.Security/KeyChain.cs

```diff
@@ -0,0 +1,94 @@
+#nullable enable
+using System;
+using System.Runtime.Versioning;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+
+namespace Android.Security
+{
+	public partial class KeyChain
+	{
+		[SupportedOSPlatform("android14.0")]
```

Comment:
Just a nitpick, but the minimum Android we support is API-21.  Thus we do not add `[SupportedOSPlatform]` attributes for anything API-21 or earlier, since they are available on "all" supported Android platforms.